specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2712 @@
+              "x-ms-mutability": [
+                "read",
+                "create"
+              ]
+            },
+            "linkedDatabases": {
+              "type": "array",
+              "description": "List of database resources to link with this database",
+              "items": {
+                "$ref": "#/definitions/LinkedDatabase"
+              }
+            }
+          }
+        },
+        "redisVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of Redis the database is running on, e.g. '6.0'"
+        },
+        "deferUpgrade": {
+          "type": "string",
+          "description": "Option to defer upgrade when newest version is released - default is NotDeferred. Learn more: https://aka.ms/redisversionupgrade",
+          "enum": [
+            "Deferred",
+            "NotDeferred"
+          ],
+          "x-ms-enum": {
+            "name": "DeferUpgradeSetting",
+            "modelAsString": true
+          }
+        },
+        "accessKeysAuthentication": {
+          "type": "string",
+          "description": "Access database using keys - default is enabled. This property can be Enabled/Disabled to allow or deny access with the current access keys. Can be updated even after database is created.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "x-ms-enum": {
+            "name": "accessKeysAuthentication",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DatabaseList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Database"
+          },
+          "description": "List of databases"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The URI to fetch the next page of results."
+        }
+      },
+      "description": "The response of a list-all operation."
+    },
+    "AccessPolicyAssignmentProperties": {
+      "title": "Redis Enterprise database access policy assignment properties",
+      "description": "Properties of Redis Enterprise database access policy assignment.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "Current provisioning status of the access policy assignment."
+        },
+        "accessPolicyName": {
+          "type": "string",
+          "description": "Name of access policy under specific access policy assignment.",
+          "default": "default",
+          "pattern": "^([a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|[a-zA-Z0-9])$",
+          "x-ms-mutability": [
+            "read",
+            "update"
+          ]
```

Comment:
wouldn't this property be required for creates?

Reply:
I guess where I'm going with this is its much simpler for our system and for customers if they don't have to deal with the possibility that meaningless access policy assignments exist.

Reply:
it has fixed name, "default" for now.